openedx/core/djangoapps/content_tagging/tests/test_objecttag_export_helpers.py

```diff
@@ -441,7 +441,7 @@ def test_build_library_object_tree(self) -> None:
         """
         Test if we can export a library
         """
-        with self.assertNumQueries(8):
+        with self.assertNumQueries(11):
```

Comment:
My [initial changes to `LibraryXBlockMetadata.from_component`](https://github.com/openedx/edx-platform/pull/35103/commits/cf9ab01300a8ba30ab037355af46be1bdf7597b1#diff-a760c3e87f2f671ac2453abbbaf6f10ca052c7acf1c46c1d36f24646e1d52a00R219) brought [this query count up to 14](https://github.com/openedx/edx-platform/actions/runs/9855392496/job/27210174606?pr=35103), because it used `component.versioning.latest`.

[Using `component.versioning.draft` instead brought the query count down to 11](https://github.com/openedx/edx-platform/pull/35103/commits/5f71cc3074dfd6024079b06a2279ab6208065a0d#diff-a760c3e87f2f671ac2453abbbaf6f10ca052c7acf1c46c1d36f24646e1d52a00R219), because the authoring API doesn't give us a more efficient way to fetch the `published_at` date than from the `last_publish_log`.

Welcome any advice on how to avoid the increased query count here!

Reply:
This seems like an @ormsbee question if I'm correctly understanding that it's about efficiently fetching modified dates from learning core's `authoring` API.